templates/webapi/test/infopanel.html.ep

```diff
@@ -39,28 +39,28 @@
                             % }
                         </div>
 
-                        Result: <b><%= $job->result %></b>
+                        Result: <b><%= $job->result %></b>\
                     % }
                     % else {
-                        State: <b><%= $job->state %></b>
+                        State: <b><%= $job->state %></b>\
                         % if (scalar($job->gru_dependencies) > 0) {
                             % my $grutext = '';
                             % for my $grudep ($job->gru_dependencies) {
                                 % $grutext .= join(",", $grudep->gru_task->id, $grudep->gru_task->taskname);
                                 % $grutext .= " ";
                             % }
-                            waiting for background (GRU) tasks: <%= $grutext %>
+                            <%= ", waiting for background (GRU) tasks: $grutext" =%>
```

Comment:
```suggestion
                            <%= ", waiting for background (GRU) tasks: $grutext" %>
```
The same counts for the other occurrences.

It apparently doesn't cause any harm but I suppose the correct way (and how we do it everywhere else, and how it is [documented](https://docs.mojolicious.org/Mojo/Template#SYNOPSIS)) is not to have the equal sign at the end.

Reply:
Oh, this *is* for trimming the surrounding white-space characters. So never mind.